jerry-core/ecma/builtin-objects/ecma-builtin-math.c

```diff
@@ -14,6 +14,7 @@
  */
 
 #include <math.h>
+#include <stdlib.h>
```

Comment:
If I'm right, `stdlib.h` comes from [this]( https://github.com/jerryscript-project/jerryscript/blob/master/jerry-core/jrt/jrt-libc-includes.h#L23.)  header, which will be included later.

Reply:
Ah, you're right, thanks!
The problem is in our `stdlib.h` header, it's missing `rand`.